po/Makefile.am

```diff
@@ -67,7 +67,7 @@ po/cockpit.c.pot:
 # Extract translate attribute, Glade style, angular-gettext HTML translations
 po/cockpit.html.pot: package-lock.json
 	$(AM_V_GEN) $(MKDIR_P) $(dir $@) && \
-	$(srcdir)/tools/missing $(srcdir)/po/html2po -d $(srcdir) -o $@ \
```

Comment:
Do we use this script anywhere else or we can drop it as well?

Reply:
Good point, these two are the last usages.

Reply:
Actually, tools/missing is generated dynamically by autoconf, not tracked in our git.